src/ruby/ext/grpc/extconf.rb

```diff
@@ -62,20 +64,32 @@
 ENV['ARCH_FLAGS'] = '-arch i386 -arch x86_64' if RUBY_PLATFORM =~ /darwin/
 ENV['CFLAGS'] = '-DGPR_BACKWARDS_COMPATIBILITY_MODE'
 
-output_dir = File.expand_path(RbConfig::CONFIG['topdir'])
-grpc_lib_dir = File.join(output_dir, 'libs', grpc_config)
-ENV['BUILDDIR'] = output_dir
+if ENV['PREFIX'] == '/data/data/com.termux/files/usr'
```

Comment:
Why not just `if ENV['PREFIX']`? no need to check the path

Reply:
Yeah but linux has different $PREFIX than termux, so just to be accurate...

BTW this one should work right?

Reply:
are you sure Linux has a $PREFIX?
You could also check $TERMUX_PREFIX

Reply:
Oh i see..i guess linux has $PATH not $PREFIX